test/integration_test/basic_test.go

```diff
@@ -45,18 +45,16 @@ var testStorageClusterBasicCases = []types.TestCase{
 		TestFunc: BasicInstall,
 	},
 	{
-		TestName:        "InstallInCustomNamespaceWithShiftedPort",
+		TestName:        "InstallInCustomNamespaceWithShiftedPortAndAllComponents",
 		TestrailCaseIDs: []string{"C52411", "C52430", "C53572"},
 		TestSpec: func(t *testing.T) interface{} {
-			cluster := &corev1.StorageCluster{
-				ObjectMeta: metav1.ObjectMeta{
-					Name:      "custom-ns-shifted-port",
-					Namespace: "custom-namespace",
-				},
-				Spec: corev1.StorageClusterSpec{
-					StartPort: func(val uint32) *uint32 { return &val }(17001),
-				},
-			}
+			objects, err := ci_utils.ParseSpecs("storagecluster/storagecluster-with-all-components.yaml")
+			require.NoError(t, err)
+			cluster, ok := objects[0].(*corev1.StorageCluster)
+			require.True(t, ok)
+			cluster.Name = "custom-ns-shifted-port-all-components"
+			cluster.Namespace = "custom-namespace"
+			cluster.Spec.StartPort = func(val uint32) *uint32 { return &val }(17001)
```

Comment:
nit: could simply create a bool pointer without function and looks more clear, but I'm ok with this

Reply:
This is how it was previously done `StartPort: func(val uint32) *uint32 { return &val }` so I just left it